core/src/main/java/org/bitcoinj/script/ScriptBuilder.java

```diff
@@ -478,6 +478,16 @@ public static Script createP2WPKHOutputScript(ECKey key) {
         return createP2WPKHOutputScript(key.getPubKeyHash());
     }
 
+    /**
+     * Creates a scriptPubKey that sends to script calculated by the given key. Read
+     * <a href="https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki">BIP 16</a> to learn more about this
+     * kind of script.
+     */
```

Comment:
The method name `createP2SHOutputScript`  has been used to adjust to the current convention name.
There are 8 methods in this class with this name. So I will leave this method name.

Reply:
There are 3 methods in this class with the existing name. 2 of those create output scripts for any (i.e. non-segwit multisig, arbitrary P2SH, or P2SH-wrapped segwit) P2SH. The third is non-segwit multisig specific (and perhaps should be renamed.)

I think this new method needs to be clearly marked as for P2SH-P2WPKH as this is a special case. At the very least the comments need to explain this, and I think it would be a good idea for the method name to be something like: `createP2SHWrappedSegwitOutputScript`,